package.json

```diff
@@ -10,7 +10,8 @@
     "test-wd": "mochify --wd test/client/**/*.spec.js",
     "test-cover": "mochify --cover test/client/**/*.spec.js",
     "test": "npm run phantom && npm run wd && npm run cover",
-    "test-server": "nodemon --watch test/server/**/*.js --watch sync-server.js --watch lib/sync-server.js test/server/test-server.js | bunyan"
+    "test-server": "nodemon --watch test/server/**/*.js --watch sync-server.js --watch lib/sync-server.js test/server/test-server.js | bunyan",
+    "eslint": "grunt"
```

Comment:
> I think if we are adding a eslint npm command, it should be grunt eslint instead of the default.

I was not really sure if we should be adding a separate npm command for eslint or not. With the other RainCatcher modules I've just added grunt to the test npm target/command which I could do in this case as well. I've updated this PR with two commits and perhaps we can have a quick chat later today to clear this up?